### ISSUE TITLE
Run Code Analysis Java Runtime error

### ISSUE THREAD

**Issue by ventil8** [vanderby/SonarQube-AzureAppService#84]
install all worked well and in my pipeline for the SonarQubeAnalyze@5 task i get the following?

##[error]java.lang.UnsupportedClassVersionError: org/sonar/batch/bootstrapper/EnvironmentInformation has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

**Comment by vanderby** [comment]
I would check the version of the Java runtime in your web app. It's under Configuration -> General -> Java Version. I see brand new deployments use Java 17.

**Comment by ventil8** [comment]
Yes uts set to Java 17 and it is a brand new deployment.  Do I need to chnage this?
